tests/Unit/ProjectCodeTest.php

```diff
@@ -60,6 +61,10 @@ public function testProductionClassesHaveUnitTests()
                 Http\Form\ForgotForm::class,
                 Http\Form\ResetForm::class,
                 Http\View\TalkHelper::class,
+                Infrastructure\Event\AuthenticationListener::class,
+                Infrastructure\Event\ExceptionListener::class,
```

Comment:
If i'm correct this list contains classes that don't have tests. Don't these have tests?